stream_alert/apps/_apps/README.rst

```diff
@@ -7,6 +7,14 @@ To obtain the access key and access key secret, an authorized user of the Aliyun
 
 Additionly, the user for whom the access key was created must have sufficient privileges to make use of ActionTrail; follow the directions on the `Grant ActionTrail permissions to RAM users <https://www.alibabacloud.com/help/doc-detail/28818.htm>`_ page.
 
+How to set up the intercom app
+###########################
+
+The Intercom API requires an access token. Get an access token by following `these instructions <https://developers.intercom.com/building-apps/docs/authorization#section-how-to-get-an-access-token>`.
+
+To specify an API version, follow `these instructions <https://developers.intercom.com/building-apps/docs/api-versioning>` to do so through Intercom's Developer Hub. 
```

Comment:
This link is also missing a trailing underscore, breaking the rendering (similar to aforementioned issue).